core/packages/PackageDB.cc

```diff
@@ -138,7 +138,7 @@ NameRef PackageDB::lookupPackage(NameRef pkgMangledName) const {
 
 const PackageInfo &PackageDB::getPackageForFile(const core::GlobalState &gs, core::FileRef file) const {
     ENFORCE(frozen);
-    auto &fileData = file.data(gs);
+    auto &fileData = file.dataAllowingUnsafe(gs);
```

Comment:
What do you think about dropping a rationale for using `dataAllowingUsafe` here, something as short as "We're only looking at the filename, which is always available.  See https://github.com/sorbet/sorbet/pull/5291 for more details."?

Reply:
Seems pretty reasonable to me!